pyaerocom/web/aerocom_evaluation.py

```diff
@@ -899,6 +941,27 @@ def find_obs_matches(self, name_or_pattern):
                            .format(name_or_pattern, list(self.obs_config.keys())))
         return matches
 
+    def _check_and_get_iface_names(self):
+        obs_list = list(self.obs_config)
+        iface_names = []
+        for obs_name in obs_list:
+            try:
+                if self.obs_config[obs_name]['web_interface_name'] == None:
+                    self.obs_config[obs_name]['web_interface_name'] = obs_name
+                else:
+                    pass
+            except KeyError:
+                self.obs_config[obs_name]['web_interface_name'] = obs_name
+            if not isinstance(self.obs_config[obs_name]['web_interface_name'],str):
+                raise ValueError('Invalid value for web_iface_name in {}. Need str type'.format(obs_name))
+            iface_names.append(self.obs_config[obs_name]['web_interface_name'])
+        iface_names = set(iface_names)
```

Comment:
@hansbrenna is there a particular reason why you convert the list to a `set`?

Reply:
I think this was discussed in an earlier pull request, and the reason was to have only unique values of iface_names.

Reply:
@jgliss @hansbrenna 
Should we ensure that this is still a list?
```suggestion
        iface_names = list(set(iface_names))
```

Reply:
> @jgliss @hansbrenna
> Should we ensure that this is still a list?

I think yes!